spec/spec_helper_acceptance_local.rb

```diff
@@ -0,0 +1,190 @@
+      run_shell('apt-get update -y')
+      # run_shell('apt-get upgrade -y')
+      run_shell('apt-get install -y lsb-release')
+      run_shell('apt-get install -y net-tools')
+    end
+
+    run_shell('puppet module install puppetlabs-stdlib --version 4.24.0', expect_failures: true)
+    run_shell('puppet module install puppetlabs-apt --version 4.4.1', expect_failures: true)
+    run_shell('puppet module install puppetlabs-translate --version 1.0.0', expect_failures: true)
+    run_shell('puppet module install puppetlabs-powershell --version 2.1.5', expect_failures: true)
+    run_shell('puppet module install puppetlabs-reboot --version 2.0.0', expect_failures: true)
+
+    # net-tools required for netstat utility being used by some tests
+    if os[:family] == 'redhat' && os[:release].to_f == '7'
+      run_shell('yum install -y net-tools device-mapper')
+    end
+
+    docker_compose_content_v3 = <<-EOS
+version: "3.4"
+x-images:
+  &default-image
+  alpine:3.8
+services:
+  compose_test:
+    image: *default-image
+    command: /bin/sh -c "while true; do echo hello world; sleep 1; done"
+      EOS
+    docker_compose_override_v3 = <<-EOS
+version: "3.4"
+x-images:
+  &default-image
+  debian:stable-slim
+services:
+  compose_test:
+    image: *default-image
+    command: /bin/sh -c "while true; do echo hello world; sleep 1; done"
+        EOS
+    docker_stack_override_v3 = <<-EOS
+version: "3.4"
+x-images:
+  &default-image
+  debian:stable-slim
+services:
+  compose_test:
+    image: *default-image
+    command: /bin/sh -c "while true; do echo hello world; sleep 1; done"
+        EOS
+    docker_compose_content_v3_windows = <<-EOS
+version: "3"
+services:
+  compose_test:
+    image: winamd64/hello-seattle
+    command: cmd.exe /C "ping 8.8.8.8 -t"
+networks:
+  default:
+    external:
+      name: nat
+      EOS
+    docker_compose_override_v3_windows = <<-EOS
+version: "3"
+services:
+  compose_test:
+    image: winamd64/hello-seattle:nanoserver
+    command: cmd.exe /C "ping 8.8.8.8 -t"
+networks:
+  default:
+    external:
+      name: nat
+      EOS
+    docker_compose_override_v3_windows2016 = <<-EOS
+version: "3"
+services:
+  compose_test:
+    image: winamd64/hello-seattle:nanoserver-sac2016
+    command: cmd.exe /C "ping 8.8.8.8 -t"
+networks:
+  default:
+    external:
+      name: nat
+      EOS
+    docker_stack_content_windows = <<-EOS
+version: "3"
+services:
+  compose_test:
+    image: winamd64/hello-seattle
+    command: cmd.exe /C "ping 8.8.8.8 -t"
+      EOS
+    docker_stack_override_windows = <<-EOS
+version: "3"
+services:
+  compose_test:
+    image: winamd64/hello-seattle:nanoserver
+      EOS
+    docker_stack_override_windows2016 = <<-EOS
+version: "3"
+services:
+  compose_test:
+    image: winamd64/hello-seattle:nanoserver-sac2016
+      EOS
+    if os[:family] == 'windows'
+      create_remote_file(host, '/tmp/docker-compose-v3.yml', docker_compose_content_v3_windows)
```

Comment:
Forgive me if you already explained this - I seem to recall a conversation about it. 🙇 

Should we be creating files in `/tmp` on Windows hosts. My memory is fuzzy but you may have mentioned in stand-up that this is not being invoked for now?

Reply:
the paths should be translated